src/FakeItEasy/Configuration/IPropertySetterConfiguration.cs

```diff
@@ -4,11 +4,11 @@ namespace FakeItEasy.Configuration
     /// Provides configuration for property setters and allows the user to specify validations for arguments.
     /// </summary>
     public interface IPropertySetterConfiguration :
-        IExceptionThrowerConfiguration,
+        IExceptionThrowerConfiguration<IPropertySetterConfiguration>,
         ICallbackConfiguration<IPropertySetterConfiguration>,
-        ICallBaseConfiguration,
+        ICallBaseConfiguration<IPropertySetterConfiguration>,
         IAssertConfiguration,
-        IDoNothingConfiguration
+        IDoNothingConfiguration<IPropertySetterConfiguration>

```

Comment:
Not in this case; we're not declaring `IDoNothingConfiguration<out TInterface>` here, just stating that `IPropertySetterConfiguration` inherits from it. (and `IDoNothingConfiguration<out TInterface>`  does have the `out`).

Anyway, I'm not sure how useful it is to declare the fluent interfaces as covariant. We do it because R# says we can, but I don't think we ever take advantage of it.